maas-schemas-ts/test/booking-create.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { Request } from '../src/tsp/booking-create/request';
+import { Response } from '../src/tsp/booking-create/response';
+
+import { decode } from 'io-ts-promise';
+import { readFileSync } from 'fs';
+
+describe('Check booking create payloads', () => {
+  describe('taxi', () => {
+    it('booking-create-request.json', () => {
+      return decode(
```

Comment:
Perhaps the test should expect promise to resolve to payload. This shouldn't technically matter but provides documentation about the intended way of using io-ts-promise.